server/protocols/webadmin-cli/src/test/resources/smtpserver.xml

```diff
@@ -0,0 +1,105 @@
+<?xml version="1.0"?>
```

Comment:
Can this file be ommitted?

Reply:
If i delete these xml files, my inmemory james/unit test will be failed. I don't know exactly which files to delete. Maybe I need some help on this?